src/data/read-through-data-cache.ts

```diff
@@ -157,8 +233,11 @@ export class ReadThroughDataCache implements ContiguousDataSource {
         (await this.contiguousDataIndex.getDataAttributes(id));
 
       if (attributes?.hash !== undefined) {
-        this.metadataStore.set(attributes.hash, {
-          accessTimestampMs: Date.now(),
+        const { arnsName, arnsBasename } = requestAttributes ?? {};
+        await this.updateMetadataCache({
```

Comment:
intended `await` here

Reply:
@dtfiedler No, good call out. No need to `await` here.